ohmyk8s.yaml

```
@@ -118,7 +118,7 @@
       src: /tmp/sops
       remote_src: yes
       dest: /usr/local/bin/
-      mode: u+x
+      mode: uo+x
```

What about this to allow the user group too?
```suggestion
      mode: a+x
```

---

@hammady should this be applied to other binaries' mode?

---

Yes, this is what I originally meant.

---

Done.